### ISSUE TITLE
Duda test 20 EffectsPart1

### ISSUE THREAD

**Issue by Sebah01a** [RodrigoToroIcarte/IIC2113-2023-1#216]
Hola, estoy tratando de modelar la carta Austin Elbow Smash, que solo puede ser jugada después de un maneuver que hizo >=5 de daño. No entiendo por qué en este test, no se considera la carta para jugarla, pues según lo que entiendo, la carta anterior fue Chair Shot (maneuver) y e hizo 8 de daño.
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/89041986/139cbba1-0ce8-444e-8367-ed2fcf53625d)
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/89041986/e6826ab0-6540-4a96-a810-8b914b64940d)

**Comment by RodrigoToroIcarte** [comment]
Lo que pasa es que la precondición se refiere a que el jugador haya hecho 5 o más de daño. En el ejemplo, el Stone Cold recibió 8 de daño. Por eso no puede jugar su carta. Si él hubiera hecho los 8 de daño, no tendría problema en usar el `Austin Elbow Smash`.

**Comment by Sebah01a** [comment]
¿Entonces, por lo que entiendo, Austin Elbow Smash nunca podrá ser jugada en la primera jugada de un turno? Ya que todavía no hace daño. Es decir, esos >=5 de daño deben ser hechos por el jugador para poder usar la carta en la siguiente jugada.

**Comment by RodrigoToroIcarte** [comment]
Exacto. Es una carta que nunca puede ser jugada como la primera carta del turno (porque requiere hacer 5D o más justo antes... en ese mismo turno).

**Comment by Sebah01a** [comment]
¡Perfecto muchas gracias!
